banbot.js

```diff
@@ -125,40 +127,33 @@ bot.on('message', async ( message ) => {
         else if ( player.tier == 4 )
             msg += '14 day suspension. '
         else if ( player.tier == 5 )
-            msg += '21 day suspension. '
+            msg += '30 day suspension. '
         else if ( player.tier == 6 )
             msg += '30 day suspension. '
-        else if ( player.tier > 6 ) {
+        else if ( player.tier >= 6 ) {
             let displayName = leftServer ? await mongo.getDisplayName( id ) : target.displayName
             msg += displayName + ' banned from server. '
         }
 
-        if ( player.tier < 7 ) {
-            if ( !leftServer ) {
-                if ( target.roles.has( suspendedId ) )
-                    msg += 'This is in addition to your current suspension. '
-                else
-                    target.addRole( suspendedId )
-                if ( target.roles.has( goldStar ) )
-                    target.removeRole( goldStar )
-            }
-            msg += '\n**ENDS:** ' + player.ends + '.'
```

Comment:
You're no longer sending when a quit suspension ends... is that intended? 

EDIT: I see the others don't really have it removed, so this feels unintentional to me. If you remove from one, I think you should remove from all, but I personally like the detailed context.